romanesco/format/__init__.py

```diff
@@ -222,14 +206,10 @@ def get_analysis(filename):
 
         for filename in validator_files:
             analysis = get_analysis(filename)
-            in_type = analysis["inputs"][0]["type"]
-            in_format = analysis["inputs"][0]["format"]
 
-            conv_graph.add_node(Validator(in_type, in_format), {
-                "type": in_type,
-                "format": in_format,
-                "validator": analysis
-            })
+            conv_graph.add_node(Validator(analysis["inputs"][0]["type"],
+                                          analysis["inputs"][0]["format"]),
+                                analysis)
```

Comment:
Would it be nicer if we had a more consistent API for this -  e.g.,  ```get_input_type(analysis)```,  ```get_input_format(analysis)```  same for outputs?